### ISSUE TITLE
[Cocoa] key callback may be called with key='a' and type=GLFW_PRESS accidentally.

### ISSUE THREAD

**Issue by haha-wei** [glfw/glfw#762]
I find this issue on my Macbook pro:
1. run a glfw window on MacOS (10.10 or 10.11)
2. switch to other application's window
3. press and hold command key, tap tab key to select glfw window, leaving the command key pressed
4. after 1 second, release command key, now we see the glfw window, but the key callback is called with key='a' and type=GLFW_PRESS.

**Comment by startas** [comment]
Yes, there is definitely something wrong with key callback - if you press and hold any keyboard button for at least a second (or something like that), then when you release it, key callback gets called at once more. Example - http://learnopengl.com/#!Advanced-OpenGL/Cubemaps , you can download this cubemaps tutorials from github, compile it and will see, that there is a few additional key callbacks after button is already released, but it happens only if you hold button pressed for at least a second (hold 2-3 seconds for guaranteed effect).

**Comment by elmindreda** [comment]
@haha-wei Unable to reproduce with the events test from GLFW 3.2 (bc649e1632d1bd9f51cc8b43bd72a8a269b090ee) on OS X 10.11.5.

**Comment by tombsar** [comment]
Can anyone reproduce this issue with the 3.3 release?
